index.html

```diff
@@ -369,7 +369,7 @@ <h4 class="date">11/5/2019</h4>
 </div>
 
 
-<p>do do do do …</p>
+<p>do do do do … la la la la la</p>
```

Comment:
@cheryl-barnes This is inaccurate.. it should be do re mi LA so ti **not** do do do la la la